app/containers/GamePage/index.js

```diff
@@ -95,17 +95,16 @@ export default class GamePage extends Component {
   }
 
   loadParticipants() {
-    const { database, participants } = this.state;
-    const { params } = this.props;
-    const usersRef = database.ref(`${params.rid}/participants`);
+    const { database, participants, rid } = this.state;
+    const usersRef = database.ref(`${rid}/participants`);
     usersRef.off();
     usersRef.limitToLast(12).on('child_added', (data) => {
       const val = data.val();
       participants.push({ key: data.key, name: val.name, role: '', health: 5, hand: [] });
       this.setState({ participants });
       usersRef.once('value', (snapshot) => {
```

Comment:
Like I mentioned earlier you don't need `usersRef.once('value'` why can't you use the same code from the 'child_removed'